packages/rx_bloc_list/lib/src/models/managed_list.dart

```diff
@@ -17,20 +17,49 @@ enum ManageOperation {
   ignore,
 }
 
+enum IncrementOperation {
```

Comment:
@StanislavPrime, what is the reason of adding sample specific models to the lib itself ?

Reply:
The sample specific models were moved to the sample project